ui/src/translations.json

```
@@ -760,7 +762,9 @@
     "Total": "Total",
     "sub flow": "Sous-flow",
     "execute": "Exécuter",
-    "execute new tab": "Exécuter dans un nouvel onglet",
+    "execute flow behaviour": "Ouverture d'une nouvelle exécution",
```

Following Anna suggestion:
```suggestion
    "execute flow behaviour": "Éxécuter le flow",
```